test/support/helpers/local.js

```diff
@@ -116,9 +116,10 @@ class LocalTestHelpers {
     return this.side.watcher.onFlush(events)
   }
 
-  startSimulation () {
+  async startSimulation () {
     return new Promise((resolve, reject) => {
       this._resolveSimulation = resolve
+      this.side.watcher.start()
```

Comment:
We probably don't want to do this: looks like a good way to end up with unexpected events.